repositories/EventRepository.ts

```diff
@@ -49,7 +49,7 @@ export class EventRepository extends BaseRepository<EventModel> {
     let query = this.repository.createQueryBuilder()
       .skip(options.offset)
       .take(options.limit)
-      .orderBy('start', 'ASC');
+      .orderBy('start', options.reverse ? 'DESC' : 'ASC');
```

Comment:
while this makes more sense, honestly lets just default to `ASC` to avoid a synchronized frontend push

Reply:
instead for [402](https://github.com/acmucsd/membership-portal-ui/pull/402) the code change would simply be to pass in the `reverse` option in the request (so we don't waste time doing the reverse, but we avoid making a breaking change)

Reply:
The ternary does default to `ASC` if `options.reverse` isn't set. And yeah, we should just pass it in as part of the request but I think it makes more sense for the past events endpoint to default to `DESC` anyways.